packages/nextlove/src/edge-helpers.ts

```diff
@@ -0,0 +1,46 @@
+import { NextRequest, NextResponse } from "next/server"
+
+export type NextloveResponse = ReturnType<typeof getResponse>
+export type NextloveRequest = NextRequest & {
+  responseEdge: NextloveResponse
+}
+
+const DEFAULT_STATUS = 200
+
+/*
+ * This is the edge runtime version of the response object.
+ * It is a wrapper around NextResponse that adds a `status` method
+ */
+export const getResponse = (req: NextloveRequest, {
+  addIf,
+  addOkStatus
+}: {
+  addIf?: (req: NextloveRequest) => boolean
+  addOkStatus?: boolean
```

Comment:
I couldn't find a way to put the addOkStatus in the exception middleware, so I put it here and it's working fine